CLI_TEST_CHECKLIST.md

```
@@ -0,0 +1,302 @@
+### Cleanup
+- [ ] `docker rm -f meshtasticd-test`
+
+### Known Simulator Limitations
+- **Reboot crash**: meshtasticd simulator uses `execv()` to reboot, which fails in Docker containers
+- **No BLE**: Simulator doesn't support Bluetooth
+- **No GPIO**: Simulator doesn't have real GPIO pins
+- **No shutdown**: Simulator can't actually power off
+
+### Docker Wrapper Fix
+The `execv()` crash is handled by wrapping meshtasticd in a restart loop:
+```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)